resources/keywords/suite_and_test_teardown.resource

```diff
@@ -0,0 +1,57 @@
+*** Settings ***
+Resource    ../keywords/login.resource
+
+
+*** Keywords ***
+Remove Provisioned DBaaSInstances
+    IF    "${instanceName}" != "${EMPTY}"
+        ${cmd_instances} =    Catenate    oc delete DBaaSInstance/${instanceName}
+        ...    --wait=true --timeout=10s --ignore-not-found
+        IF    """dev""" in """${instanceName}""" and "${newProject}" != "${EMPTY}"
+             ${cmd_instances} =    Catenate    ${cmd_instances}    -n ${newProject}
+        END
+        ${stdout} =   Execute Command    ${cmd_instances}    True
+    END
+    Sleep    7s
+    Comment    Blind Sleep provided to let DB Cluster get deleted on the ISV Side
+    ...        before the associated DBaaSInventory is deleted from RHODA
+    ...        If the associated DBaaSInventory(Provider Account) is deleted from RHODA
+    ...        before the database cluster gets deleted from ISV side, the concerned
+    ...        DBaaSInstance CRD will eventually get deleted but the DB Cluster will not
+    ...        get removed on ISV side
+
+Remove Created Namespaces
+    IF    "${newProject}" != "${EMPTY}"
+        ${cmd_namespaces} =    Catenate    oc delete ns/${newProject} --wait=True --timeout=10s --ignore-not-found
+        ...    && echo "Namespace deleted within timeout" || echo "Could not delete Namespace, will be force deleted"
+        ${stdout} =    Execute Command    ${cmd_namespaces}    True
+    END
+
+Remove Imported Provider Accounts
+    IF    "${provaccname}" != "${EMPTY}"
+        ${cmd_provider_account} =   Set Variable    oc delete DBaaSInventory/${provaccname} -n redhat-dbaas-operator --wait=true
+        ${stdout} =    Execute Command    ${cmd_provider_account}    True
+    END
+
+Force Delete Terminating Namespaces
```

Comment:
IMHO force deleting namespace which are stuck in termination can be done manually and not in favor of keeping it on part of our framework. Please LMKWYT?

Reply:
OK sure.